src/fhirtypes/primitiveTypes.ts

```diff
@@ -0,0 +1,11 @@
+import { InvalidDateTimeError } from '../errors/InvalidDateTimeError';
+
+export type FHIRdateTime = string;
```

Comment:
Is there a reason you called this `FHIRdateTime`?  Is it because some of the other primitive names would clash with reserved words in JavaScript, so you're establishing the pattern now?

I think that's a valid reason, but I would have expected the D to be capitalized (e.g., `FHIRDateTime`).  We already have an example of this with our `FHIRDefinitions` class.